app/screens/home/channel_list/additional_tablet_view/additional_tablet_view.tsx

```diff
@@ -28,10 +31,17 @@ const ComponentsList: Record<string, React.ComponentType<any>> = {
 };
 
 const channelScreen: SelectedView = {id: Screens.CHANNEL, Component: Channel};
-const globalScreen: SelectedView = {id: Screens.GLOBAL_THREADS, Component: GlobalThreads};
+const globalThreadsScreen: SelectedView = {id: Screens.GLOBAL_THREADS, Component: GlobalThreads};
+const globalDraftsScreen: SelectedView = {id: Screens.GLOBAL_THREADS, Component: GlobalDraftsAndScheduledPosts};
```

Comment:
Should this be `{id: Screens.GLOBAL_DRAFTS}`?

Reply:
Yes

Reply:
Made changes.